code/game/gamemodes/cult/cult_items.dm

```diff
@@ -46,6 +46,10 @@
 		else
 			user.adjustBruteLoss(rand(force/2, force))
 		return
+	if(!iscultist(target))
```

Comment:
Is this check neccessary? It means that cultist friendly fire is off, and maybe that's desireable... but maybe it's not?